### PR TITLE
Update Darc.md with note about auth

### PR DESCRIPTION
**Pull request by chcosta**
### To double check:

* [ ] The right tests are in and the right validation has happened.  Guidance: https://github.com/dotnet/arcade/blob/main/Documentation/Validation.md

### CODE REVIEW

File: Documentation/Darc.md [hunk 1]
```diff
@@ -164,6 +164,9 @@ D:\repos\arcade> darc get-channels
 .NET 3 Tools - Validation
 ```
 
+*Note: `darc get-channels` does not exercise all authentication paths which require PAT's.  If you run a darc command and see an error like `Response status code does not indicate success: 401 (Unauthorized)`, it likely indicates that you are missing an AzDo or GH PAT (or that one of those PAT's is invalid / expired).*
```
Comment (mmitche) [inline Documentation/Darc.md:167]: AzDO pats aren't required these days, since they use your local creds by default. A GH pat is the likely culprit.

**Review by radical — APPROVED** [review]

File: Documentation/Darc.md (hunk 1 above)
Comment (chcosta) [reply Documentation/Darc.md:167]: Does that mean we should deprecate the bullet point above about the AzDo PAT as well?

> An Azure DevOps PAT for downloading files from Azure DevOps

Comment (chcosta) [reply Documentation/Darc.md:167]: @mmitche ?

Comment (rmarinho) [reply Documentation/Darc.md:167]: I still had to create one for access internal feeds.

**Pull request merged**